pkg/storage/sqlstorage/accounts.go

```diff
@@ -133,18 +133,23 @@ func (s *Store) getAccounts(ctx context.Context, exec executor, q storage.Accoun
 	}
 
 	var previous, next string
-	if q.Offset-q.Limit > 0 {
-		t.Offset = q.Offset - q.Limit
+	if q.Offset > 0 {
+		offset := int(q.Offset) - int(q.PageSize)
+		if offset < 0 {
+			t.Offset = 0
+		} else {
+			t.Offset = uint(offset)
```

Comment:
is this line hard to reproduce for testing ?

Reply:
I don't think it is hard. but i don't think it is really helpful. This line will never fail, it is just a simple assignment.
I can add a test if you think it is required.